config/README.rst

```diff
@@ -237,21 +242,35 @@ with-sphinx-doctests
 Coverage options
 ````````````````
 
+The corresponding section is named: ``[coverage]``.
+
 fail-under
   A minimal value of code coverage below which a test failure is issued.
 
 
 Coverage:run options
 ````````````````````
 
+The corresponding section is named: ``[coverage-run]``.
```

Comment:
I know, it does not belong to this PR, but I believed, that nested structures were one of the selling point of toml. I would have expected the configuration for `coverage run` under `[coverage.run]`. But that requires of course some change of code.

Reply:
I thought this would make the configuration file unnecessary complex and I do not really like the syntax (and its variants) used for nested structures.